lints/lint_ext_cert_policy_explicit_text_too_long.go

```diff
@@ -50,7 +50,13 @@ func (l *explicitTextTooLong) CheckApplies(c *x509.Certificate) bool {
 func (l *explicitTextTooLong) Execute(c *x509.Certificate) *LintResult {
 	for _, firstLvl := range c.ExplicitTexts {
 		for _, text := range firstLvl {
-			if len(text.Bytes) > 200 {
+			var runes string
+			if text.FullBytes[0] == 30 {
```

Comment:
A brief comment here would be helpful, laying out the intent and any assumptions -- I can sort of infer that `text.FullBytes` is the encoded (assumed to be / guaranteed to be DER?) BMPString, with tag 0x1e == 30, but it's definitely not clear from the context alone. 

Also, it should probably use constant for the tag rather than the magic number 30; but even then, doing a straight `tag == BMP_STRING_0x1E` comparison may fail for weird corner cases like CONSTRUCTED strings (though off-hand, I'm not sure that's a valid scenario).

Reply:
I added a short comment here indicating what the Bytes we are inspecting are. I also switched to using a constant defined at the top of the lint just so it would be clear what Tag we are looking at and compare to the Tag instead of the FullBytes.